### ISSUE TITLE
Re-revert #13357

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [containers/podman#13393]
This PR: https://github.com/containers/podman/pull/13357 was inadvertently merged into the v4.0 branch when it was supposed to be frozen for all but critical fixes as we're readying for a RHEL 8.6 and 9.0 release.  That was reverted through #13392.  Once the v4.0 branch is opened for non-critical bugs again, or when a new branch is made ready for RHEL 8.6.1 and RHEL 9.0.1, then #13357 should be cherry-picked into it again.

@Romain-Geissler-1A FYI

**Comment by Romain-Geissler-1A** [comment]
Hi,

Ok, I acknowledge that.

I tried to ask this several times in issues in the past: would it be possible for Red Hat to ship a "upstream" stream for the "container-tools" module in RHEL ? Obivously, it would be shipped without any Red Hat support: use it at your own risk, and don't contact Red Hat paying support if it doesn't work. Podman is a tool that is still a young project moving fast upstream, which somehow goes in the opposite direction of the need for stability in RHEL (where podman is updated every 3 months). I often find myself willing to use the "next" podman that RHEL doesn't ship yet.

Cheers,
Romain

**Comment by TomSweeneyRedHat** [comment]
@fatherlinux is our Product Manager.  Scott, this PR was merged just after the door closed for RHEL 8.6 and the next delivery vehicle would be 8.6.1 a few weeks after that.  IDK we ship anything upstream in RHEL, would an answer be CentOS stream or something else?

**Comment by Luap99** [comment]
This was reverted in https://github.com/containers/podman/pull/13392

**Comment by Luap99** [comment]
Sorry you are right. Double negative is always confusing.

@TomSweeneyRedHat Since we now have a 4.0-rhel branch this must go there instead if it should end up in RHEL with podman 4.0.

**Comment by Romain-Geissler-1A** [comment]
Will this be backported on 4.0.x only for RHEL, or all 4.0.x version, also on other distros ? On client side (where this pull request has an effect), we use sometimes RHEL, and sometimes Ubuntu/Debian. Now ubuntu 22.04 has an official podman packages (not yet updated to podman 4, but it will be one day).

**Comment by TomSweeneyRedHat** [comment]
@Romain-Geissler-1A The plan is to get this into Podman v4.0.*, probably v4.0.4, both in RHEL and the other distributions.  It will also be in Podman v4.1 in all distributions other than RHEL.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @Luap99 what is going on with this issue?

**Comment by TomSweeneyRedHat** [comment]
I just made a note in the cherry pick on the v4.0 branch.  That PR can be merged at any time @mheon is comfortable doing so.  The fix is in upstream already, and that is targeted for Podman v4.1, which will be included in RHEL 8.6.0.2.  So once https://github.com/containers/podman/pull/13395 is merged, this can be closed.

**Comment by vrothberg** [comment]
Friendly ping. What's the current status of this issue?

**Comment by Luap99** [comment]
https://github.com/containers/podman/pull/13395 is merged so I think we can close it.
